cypress/e2e/token.test.ts

```diff
@@ -0,0 +1,20 @@
+import { getTestSelector, getTestSelectorStarsWith } from '../utils'
+
+describe('Testing nfts', () => {
+  before(() => {
+    cy.visit('/')
+  })
+
+  it('should load nft leaderboard', () => {
+    cy.visit('/tokens/ethereum')
+    cy.get(getTestSelectorStarsWith('token-table')).its('length').should('be.gte', 25)
+  })
+
+  it('should load go to ethereum token and return to token list page', () => {
+    cy.visit('/tokens/ethereum')
+    cy.get(getTestSelector('token-table-row-Ether')).first().click()
```

Comment:
you can do this just based on the text name within table instead of needing this test id